package.json

```diff
@@ -46,7 +46,7 @@
 		"is-scoped": "^2.1.0",
 		"issue-regex": "^3.1.0",
 		"listr": "^0.14.3",
-		"listr-input": "^0.2.0",
+		"listr-input": "LuKks/listr-input#0.2.1",
```

Comment:
You need to get your changes added to `listr-input` first.

Reply:
There is already a https://github.com/SamVerschueren/listr-input/pull/8 for that but the author is not merging it yet after several months. So the plan was, we can use my repo until the merge because I really need the IssueHunt. Anyway, thank you for the review.